app/utils/RegexConstants.scala

```diff
@@ -43,13 +43,16 @@ trait RegexConstants {
   final val nonApiNameRegex             = """^[a-zA-Z0-9 "'&,\-\\\/]*$"""
   final val orgNameRegex                = """^[a-zA-Z0-9 &`\-\'\\\^]*$"""
   final val utrRegex                    = "^[0-9]*$"
+  final val giinRegex                   = "^[a-zA-Z0-9]{6}\\.[a-zA-Z0-9]{5}\\.[a-zA-Z0-9]{2}\\.[a-zA-Z0-9]{3}$"
```

Comment:
Do we expect to use the GIIN regexs _anywhere_ else except for in the `mandatoryGIINFormatter` function?

If not, then I think the regex should live in the function itself. Keeps the code all in one place. I'm not convinced there's value in having it in a separate file if it's only used in one place.

Reply:
See above^